domain/machine/state/state.go

```diff
@@ -181,6 +181,12 @@ DELETE FROM net_node WHERE uuid IN
 	return errors.Annotatef(err, "deleting machine %q", mName)
 }
 
+// InitialWatchModelMachinesStatement returns the table and the initial watch
+// statement for watching life changes of non-container machines.
+func (st *State) InitialWatchModelMachinesStatement() (string, string) {
+	return "machine", "SELECT name FROM machine WHERE name NOT LIKE '%/%'"
```

Comment:
**suggestion**: Is the `NOT LIKE` only to filter container machines? If that's the reason why not adding a boolean column? it will have better performance than this query

Reply:
We should probably move to that at a later point, but this is the filter in the Mongo watcher.